libs/ml/tests/unit/ops/tanh.cpp

```diff
@@ -196,8 +200,8 @@ TYPED_TEST(TanHTest, saveparams_backward_all_negative_test)
 
   for (uint64_t i(0); i < n; ++i)
   {
-    data.Set(i, typename TypeParam::Type(dataInput[i]));
-    error.Set(i, typename TypeParam::Type(errorInput[i]));
+    data.Set(i, fetch::math::AsType<DataType>(dataInput[i]));
+    error.Set(i, fetch::math::AsType<DataType>(errorInput[i]));
```

Comment:
errorInput and dataInput is wrong style (should be error_input and data_input)

Reply:
fixed